### ISSUE TITLE
Expose options object to allow passing in browsersync options

### ISSUE THREAD

**Issue by johnpapa** [johnpapa/lite-server#1]

**Comment by wsmd** [comment]
+1 :+1:

**Comment by festen** [comment]
+1

**Comment by johnpapa** [comment]
considering if we should expose each option in browsersync OR create an rc.config file that lite-server looks at to start up. thoughts?

I'm leaning towards the rc file.

**Comment by vikeen** [comment]
Doing an .rc file and explaining in the documentation that it links to the browser-sync configuration is good by me.

**Comment by EricLease** [comment]
I had seen this github issue when trying to come up with a solution for [this question I posted on SO](http://stackoverflow.com/q/34824621/4342563).  To say I'm not active in the open source community would be an understatement.  I'm just commenting here because I just addressed this issue (locally only) for the `browser` option used by browser-sync.  I don't really care if an .rc is used or not, but you might want to consider the fact that the Angular 2 team is using lite-server in it's demo, and certain precedents are being established by default for people using that quick start guide (I would speculate that number is growing rapidly). 

Examples:
1. `--open` is consumed and altered by `lite-server` before reaching `browser-sync` as part of the `startPath`, can cause a conflation with `browser-sync`'s `open` option.
2. Some options, such as `port` are being passed through unchanged to `browser-sync`, as the same option name
